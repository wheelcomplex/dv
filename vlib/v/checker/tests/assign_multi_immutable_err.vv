fn main() {
	a := 10
	b := 20
	a, b = 1, 2

	println('$a, $b')
}