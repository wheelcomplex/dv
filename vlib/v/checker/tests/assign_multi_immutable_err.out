vlib/v/checker/tests/assign_multi_immutable_err.vv:4:2: error: `a` is immutable, declare it with `mut` to make it mutable
    2 |     a := 10
    3 |     b := 20
    4 |     a, b = 1, 2
      |     ^
    5 |
    6 |     println('$a, $b')